triton-docker

```diff
@@ -68,11 +68,28 @@ function currentProfile () {
         local profile_source='default; use `triton profile set <profile name>` to change'
     fi
 
-    echo "Executing in '$(triton profile get | awk 'NR == 1 {print $2}')' (${profile_source})"
+    # Get and the user's account information and CloudAPI URL for parsing later
+    local triton_account="$(triton account get)"
+
+    echo "Executing in '$(echo "${triton_account}" | awk 'NR == 1 {print $2}')' (${profile_source})"
+
+    # Check if CNS is enabled, require it
+    if [ ! "true" == "$(echo "${triton_account}" | awk -F": " '/cns/{print $2}')" ]
```

Comment:
That is brittle. If I have "cns" in my email or name or other fields:

```
$ triton account get| awk -F": " '/cns/{print $2}'
trent.mick+cns-guy@joyent.com
true
```

Perhaps:

```
... | awk -F": " '/^triton_cns_enabled:/{print $2}'
```